extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueLightHandler.java

```diff
@@ -93,13 +106,28 @@ public void initialize() {
                 updateStatus(statusInfo.getStatus(), statusInfo.getStatusDetail(), statusInfo.getDescription());
                 FullLight fullLight = getLight();
                 if (fullLight != null) {
+                    String modelId = fullLight.getModelID().replaceAll(NORMALIZE_ID_REGEX, "_");
+                    updateProperty(Thing.PROPERTY_MODEL_ID, modelId);
```

Comment:
I agree for using DSLs but find this a rather limiting approach. I would suggest to change the exception thrown when read-only properties are updated to just a warning. If I would remove the updateProperty calls from the hue binding completely, we would loose functionality based on these properties (firmware updating, model identification)